src/main/java/yevhenii/lostfilmdemo/controllers/ImdbClientTest.java

```diff
@@ -0,0 +1,10 @@
+package yevhenii.lostfilmdemo.controllers;
+
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.context.annotation.Profile;
+
+@Profile("test")
+@FeignClient(url = "http://localhost:8089/www.omdbapi.com/", name = "omdb")
```

Comment:
you don't need to create separate client, you can put url into properties file and place here ${property.name}. Then just ovverider property in test properties

Reply:
idk how to use @Value in interface

Reply:
you don't need to use value interface, you should have `(url = "${omdb.api.url}")`